network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -523,7 +523,10 @@ def get_product_subset(cutoff_date, authenticated, key, products, language_code=
     if not authenticated:
         products = products.live()
 
-    products = products.prefetch_related("evaluation__votes")
+    products = products.prefetch_related(
+        "evaluation__votes",
```

Comment:
**Question:** Should the prefetch for votes be here, as it was identified as the cause of memory issues?

Reply:
I'm addressing that issue in another PR

Reply:
Got it sorry, the formatting change made me think you were adding it rather than it just being moved, my mistake!